USAGE.md

```diff
@@ -72,14 +71,15 @@ Here are some example voice commands:
  * "Alexa, turn on Bedroom Light"
  * "Alexa, turn on Bedroom Light"
  * "Alexa, dim Kitchen Lights to 30 percent"
+ * "Alexa, what is the downstairs temperature?"
+ * "Alexa, set the downstairs temperature to 72 degrees"
```

Comment:
I don't want to start a unit system fight here but I think it's common to have defaults and examples in degree Celsius rather than Fahrenheit 🙈

Reply:
Ha! Its in celsius.. Have my sauna downstairs... 😆 

Will change